week7/cooper_client_react/11_client_user_authentication.md

```diff
@@ -191,63 +192,81 @@ If you run the feature test again, you will see that the change we made has not
 We are going to add a method called `onLogin` that is going to authenticate the user against the backend system.
 
 First, you need to add this method to the App component:
-```
-async onLogin(e) {
-  e.preventDefault();
-  let resp = await authenticate(this.state.email, this.state.password)
-  if (resp.authenticated === true) {
-    this.setState({ authenticated: true });
-  } else {
-    this.setState({ message: resp.message, renderLoginForm: false })
-  }
-}
+
+```js
+// src/App.jsx
+
+...
+onLogin = async e => {
+    e.preventDefault();
+    let resp = await authenticate(this.state.email, this.state.password);
```

Comment:
please make use of the submit form event instead of storing email and password in the state

Reply:
Done